src/sentry/issues/search.py

```diff
@@ -219,8 +220,9 @@ def _query_params_for_generic(
         if categories is None:
             logging.error("Category is required in _query_params_for_generic")
             return None
-
         category_ids = {gc.value for gc in categories}
+        if referrer != "api.feedback_index":
```

Comment:
Are there other places in the codebase where we change behavior based on the referrer? To me it's a bit unexpected and I think I'd prefer a more explicit way of telling this code path you either do or don't want feedbacks. But if this is something we do elsewhere then this is fine.

Reply:
I think ideally we'd have a separate endpoint that only queried feedbacks, and feedbacks by default are hidden in this endpoint. 

This would allow us to remove all of the referrer based behavior here. I'm not sure if we modify behavior by referrer elsewhere.

I agree that it is a bit weird -- I can attempt some refactors and making a separate endpoint as a follow up and then remove the referrer logic here -- what do you think?

Reply:
The followup with separate endpoints sounds like a great solution to me!

Reply:
+1 on this feeling awkward, I'd rather see that as a specific param or different endpoint path.